api.php

```diff
@@ -801,6 +848,56 @@ public function getUserID() {
 	}
 }
 
+class ApiKey extends ApiAuth {
+
+	/**
+	 * API access key.
+	 * @var string
+	 */
+	private $apiKey;
+
+	protected function load(){
+		global $DB;
+		$rowUserId = $DB->sql_hash("SELECT userID from wD_ApiKeys WHERE apiKey = '".$DB->escape($this->apiKey)."'");
```

Comment:
is there a way to use a stored procedure here to protect against SQL injection?

Reply:
This is their code, so I don't want to make changes to the way they handle the DB, but we can bring that up as it is a valid concern.